community/planning/service_timer.md

```diff
@@ -0,0 +1,245 @@
+# Service Timer
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+## Summary
+
+A timer-based model for waking up specific services and executing code. Both
+APIs for writing service implementations as well as runtime components to run
+services are covered.
+
+## Motivation
+Service implementations execute code via two primary handler types:
```

Comment:
Typically requires a empty line after a header.  This happens with a number of headers below.

Reply:
Fixed